src/main/java/org/myrobotlab/service/Runtime.java

```diff
@@ -3458,4 +3498,39 @@ static public void setAllLocales(String code) {
     }
   }
 
+  @Override
+  public String created(String serviceName) {

```

Comment:
runtime doesn't do anything when it's created?  what's the point of this method?  if it's framework stuff and it's for publishing .. it should be called publishCreated .. publishStarted.. publishStopped  etc..

Reply:
legacy - previously it has always been register(ed) and release(ed).
I didn't want to incur too many changes.